src/components/Header/Header.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React, { FormEvent } from 'react';
+
+type Props = {
+  title: string,
+  setTitle: (value: string) => void;
+  onAdd: (event: FormEvent<HTMLFormElement>) => void;
+  activeTodosQuantity: number;
+  onUncompletedCheckedChange: () => void;
+  isLoading: boolean;
+};
+
+export const Header: React.FC<Props> = ({
+  title,
+  setTitle,
+  onAdd,
+  activeTodosQuantity,
+  onUncompletedCheckedChange,
+  isLoading,
+}) => {
+  return (
+    <header className="todoapp__header">
+      {!!activeTodosQuantity}
+      <button
+        type="button"
+        className={classNames('todoapp__toggle-all', {
+          active: true,
+        })}
+        onClick={onUncompletedCheckedChange}
+      />
```

Comment:
button should be shown only if you have activeTodosQuantity, you probably forgot to do this